src/snowflake/snowpark/dataframe.py

```diff
@@ -1813,15 +2370,18 @@ def pivot(
             snowflake.snowpark.relational_grouped_dataframe._PivotType(
                 pc[0], pivot_values, default_on_null
             ),
+            _ast_stmt=stmt,
         )
 
     @df_api_usage
+    @publicapi
     def unpivot(
         self,
         value_column: str,
         name_column: str,
         column_list: List[ColumnOrName],
         include_nulls: bool = False,
```

Comment:
The existing code doesn't capture `include_nulls`, because it's a very recent addition. We need a `TODO` for this particular item. More generally, I jumped straight from `1.23` to `HEAD` when doing the latest `main` -> `phase 0` integration, so we'll need to pass over the public API surface and ensure the AST client is still capturing everything at full fidelity.

Reply:
Yes, this has been already added to the umbrella ticket https://snowflakecomputing.atlassian.net/browse/SNOW-1826001 for post-merge consideration.